docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,382 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+arm.pc       | UINT32 | instruction pointer
+arm.lr       | UINT32 | instruction pointer
+aarch64.pc   | UINT64 | instruction pointer
+aarch64.lr   | UINT64 | instruction pointer
+sys.signal   | INT32  | signal that caused crash
+sys.tid.comm | STRING | thread name
+
+Depending on the architecture, array will contain:
+
+Arch    | PC (instruction pointer) | Link Register
+--------|--------------------------|------------------
+x86     | x86.eip                  | (not available)
+x86_64  | x86_64.eip               | (not available)
+arm     | arm.pc                   | arm.lr
+aarch64 | aarch64.pc               | aarch64.lr
+
+### minicoredumper
+
+minicoredumper is a tool that accepts a core dump on a standard input and saves the modified (size-reduced) core dump in a file. This tool is internally used by crash-manager to generate minicoredump. Modifications depends on the configuration, and they mainly rely on minimization of the core dump file.
+
+Usually, integrator does not need to modify this configuration unless integrator needs to change:
+
+-   parameters of minicoredump files
+-   for which applications minicoredump is created and for which regular core dump is created
+
+minicoredumper can be used alone by setting `/proc/sys/kernel/core_pattern` to `|/usr/bin/minicoredumper %P %u %g %s %t %h %e`. All possible `%` specifiers are described in the [core(5)](http://man7.org/linux/man-pages/man5/core.5.html)).
+
+The default path of the configuration file is `/etc/minicoredumper/minicoredumper.cfg.json` and the file format is JSON. Possible options are:
+
+-   **base_dir** (string): Defines the root directory where the dumped data will be stored. This options is overwritten by the command line option, which is set by **crash-manager**.
+-   **watch** (array): A set of recepts, which will be used for applications that meets the defined criteria:
+    -   **exe** (string): The full path to the binary returned by readlink. Can contain the `*` character to wildcard matching. If not specified, the `*` value will be used
+    -   **comm** (string): The basename of the command that was run. Can contain the `*` character to wildcard matching. If not specified, the `*` value will be used
+    -   **recept** (string): The full path to the recept file
+
+    > **Note**
+    >
+    > If **exe** and **comm** are specified, then both conditions must be met to use the recept.
+
+Default minicoredumper.cfg.json:
+
+```
+  {
+      "base_dir": "/usr/local/var/crash/minicoredumper",
+      "watch": [
+          {
+              "exe": "/usr/bin/dotnet-launcher",
+              "recept": "dotnet.recept.json"
+          },
+          {
+              "recept": "generic.recept.json"
+          }
+      ]
+  }
+```
+
+
+The default path of the recept files is `/etc/minicoredumper/` and the format is JSON. Possible options are:
+
+-   **stacks** (list): Contains settings for what should be dumped:
+    -   **dump_stack** (bool): If true, stacks will be dumped
```

Comment:
@mijin85cho,

The reason I gave this comment is to maintain consistency with the entire document. If it can be changed in the entire document then I would also suggest the same..

Reply:
@surbhig12 First, thanks for trying to check the consistency.
However I'm not sure from where you get the concept of removing period from the sub-bullet list, even to the sentence, not only to phrase.
Would you please re-consider that?

Reply:
@mijin85cho,

I too agree that we must have periods at the end of the sentence, even for sub-bullet list.
But in the existing file, they haven't used it. For example, for the content under the heading, deviced. This is the reason, I gave a global comment. 
We can ignore this comment and add a period.

Reply:
@surbhig12 
Is "the content under the heading, **deviced**" you meant is below?

![image](https://user-images.githubusercontent.com/9958374/63742576-350a4c00-c8d4-11e9-95dc-fa2e4df52eee.png)

If so, I'd say the list items are not in sentence form but in gerund form.

Anyway, let's keep adding periods at the end of the sentence.